cdap-docs/tools/docs-cli-commands.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright © 2016 Cask Data, Inc.
+#
+# Used to generate a table in the CLI documentation from the output of CLI tools.
+# 
+# If input_file is not found, puts out a warning message in the log and puts a note 
+# in the table that is generated.
+#
+
+from optparse import OptionParser
+import os
+import sys
+
+VERSION = "0.0.1"
+
+LITERAL = '``'
+SPACES = ' ' * 3
+SECTION_LINE = SPACES + '**'
+COMMAND_LINE = SPACES + LITERAL
+LITERAL_LINE = SPACES + ' | '
+
+SKIP_SECTIONS = ['security (beta)']
```

Comment:
The casing here is different than the actual section's casing.
Also, the casing has an effect on whether the section is skipped or not. I think you should make the casing consistent (change what's here), and then perhaps make the comparison below case-insensitive.